docs/proto/proto-docs.md

```diff
@@ -4,56 +4,60 @@
 
 ## Table of Contents
 
-- [hyperlane/ism/v1/ism.proto](#hyperlane/ism/v1/ism.proto)
-    - [Ism](#hyperlane.ism.v1.Ism)
+- [hyperlane/igp/v1/genesis.proto](#hyperlane/igp/v1/genesis.proto)
```

Comment:
`make proto-all` does this. I think we need separate doc locations? I get "duplicate" warnings.